### DISCUSSION TITLE
Be able to select multiple data points on data visualizations.

### DISCUSSION THREAD

**Discussion by NoSadBeHappy** [grafana/grafana#73251] [Dashboards, Panels & Visualizations]
I want to be able to select a couple of data points and compare them without showing all queries. I can choose one of them by clicking in the legend, but not multiple. Only one or all.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82556. Feel free to continue the discussion around this there. Thank you!
